src/pages/pools/new.tsx

```diff
@@ -1,71 +1,193 @@
+            setError("Please select files or supply a pool name.")
+            return
+          }
+          setLoading(true)
+          const {name, order, key, format} = formData
+          await createAndLoadFiles.run(files, {name, order, key, format})
+          setLoading(false)
+        }}
+      >
+        <Drop>
+          <Field>
+            <InputLabel htmlFor="files">Files</InputLabel>
+            <FileInput
+              multiple
+              name="files"
+              autoFocus
+              files={files}
+              setFiles={setFiles}
+              onChange={(e) => {
+                if (e.currentTarget.files.length > 0) setError("")
+              }}
+            />
+          </Field>
+          <Field>
+            <InputLabel htmlFor="format">Format</InputLabel>
+            <DataFormatSelect name="format" defaultValue={defaults["format"]} />
+          </Field>
+        </Drop>
+        <Field>
+          <InputLabel htmlFor="name">Pool Name</InputLabel>
+          <TextInput
+            name="name"
+            placeholder="Derive from files..."
+            defaultValue={defaults["name"]}
+          />
+        </Field>
+        <Field>
+          <InputLabel htmlFor="key">Pool Key</InputLabel>
           <TextInput
-            autoFocus
-            value={name}
-            onChange={(e) => setName(e.target.value)}
+            name="key"
+            placeholder="None"
+            defaultValue={defaults["key"]}
           />
-        </InputField>
-        <SubmitWrap>
-          <PrimaryButton>Create</PrimaryButton>
-        </SubmitWrap>
-      </FormWrap>
-    </Wrap>
+        </Field>
+        <Field>
+          <InputLabel htmlFor="order">Sort Order</InputLabel>
+          <SelectInput name="order" defaultValue={defaults["order"]}>
+            <option value="desc">Descending</option>
+            <option value="asc">Ascending</option>
+          </SelectInput>
+        </Field>
+        {error && <FormError>{error}</FormError>}
+        <SubmitButton
+          disabled={loading}
+          style={{
+            minWidth: "100%",
+            marginTop: 12,
+            alignSelf: "center",
+          }}
+        >
+          {loading ? "Loading..." : "Create Pool"}
+        </SubmitButton>
+      </Form>
+      <Footer>
+        <Help>
+          <b>Auto-detectable formats</b>:{" "}
+          <Link href="https://tools.ietf.org/html/rfc4180">CSV</Link>,{" "}
+          <Link href="https://www.json.org/json-en.html">JSON</Link>,{" "}
+          <Link href="http://ndjson.org/">NDJSON</Link>,{" "}
+          <Link href="https://docs.zeek.org/en/current/log-formats.html#zeek-tsv-format-logs">
+            Zeek
+          </Link>
+          , <Link href={links.ZED_DOCS_FORMATS_ZJSON}>ZJSON</Link>,{" "}
+          <Link href={links.ZED_DOCS_FORMATS_ZNG}>ZNG</Link>, and{" "}
+          <Link href={links.ZED_DOCS_FORMATS_ZSON}>ZSON</Link>.<br></br>
```

Comment:
@jameskerr: I'm guessing this use of `<br>` is inappropriate, so feel free to overhaul my formatting. 😂